package.json

```diff
@@ -76,6 +78,7 @@
     "test-prepare": "tools/test.sh prepare",
     "test-prepare-watch": "tools/test.sh prepare -w",
     "test-e2e": "tools/postgres/prepareTestDB.sh && tools/test.sh test-e2e --timeout=60s --verbose",
+    "test-screenshot": "docker-compose run screenshot-tests bash -c 'node_modules/.bin/ava app/assets/javascripts/test/tracing-view-screenshot-tests/*.screenshot.js'",
```

Comment:
I could probably move the bash part into the docker-compose file and simply call `docker-compose up screenshot-tests` here, right @jstriebel?

Reply:
Yes. Or, if one might want to run puppeteer locally, you can just have
`bash -c 'node_modules/.bin/ava app/assets/javascripts/test/tracing-view-screenshot-tests/*.screenshot.js'`
in `package.json`.
Then set the command in the docker-compose.yml to `yarn run test-screenshot` or similar.

I think the second option is most clean.